languages.csv

```diff
@@ -45,6 +45,7 @@ Snd,Sindhi,sindhi,,,,,,,,y
 Spa,Spanish,spanish,Romance,,y,,,,y,y
 Swa,Swahili,swahili,Bantu,,,y,n,n,n,n
 Swe,Swedish,swedish,Scand,,y,,,,y,y
+Tam,Tamil,tamil,,,,,,,,n
```

Comment:
Change the tamil entry into `Tam,Tamil,tamil,n,n,n,n,n,n,n,n`, so running `make install` won't try to create the API and compile the files. We'll change this to have more `y`s once the Tamil RG has more stuff in it.